### DISCUSSION TITLE
Mountain Duck disappeared

### DISCUSSION THREAD

**Discussion by Sigurd-git** [iterate-ch/cyberduck#14519] [Q&A]
This ticket system is to report bugs and feature requests. For support, visit the [help page](https://cyberduck.io/help/) first.

 * If you have trouble connecting to a server or your login credentials are not valid, try to resolve the issue with the assistance of your hosting service provider. Any such ticket will be closed with a resolution of _thirdparty_.

**Describe the bug**
I just downloaded Mountain duck and connect it to google drive. Then the Mountain duck disappears in the foreground. I can see it work in the background through the activity monitor. I killed it and reopened Mountain duck, but it did not work. Then I unmounted google drive, but Mountain duck did not appear. Restart the computer did not help.  
From MacBook pro 14.

**Screenshots**
<img width="740" alt="image" src="https://user-images.githubusercontent.com/52092818/231390359-0fc8cb19-cbd1-4ec9-afe4-bf513ece2642.png">

**Desktop (please complete the following information):**
 - OS: macOS Ventura 13.3.1

**Comment by dkocher** [comment]
Mountain Duck runs in the status bar (Mac) and taskbar (Windows). Please refer to our online [documentation](https://docs.mountainduck.io/mountainduck/interface/#user-interface) available. If you have many status bar items, the Mountain Duck icon may not show when there is not enough space available between the notch and the right edge of the display. Please try if Mountain Duck shows up if you quit some of the currently running status bar applications or remove status bar icons not in use.
